src/main/java/com/elastic/support/diagnostics/commands/RunClusterQueriesCmd.java

```diff
@@ -24,7 +26,8 @@ public void execute(DiagnosticContext context) {
 
         try {
             DiagConfig diagConfig = context.getDiagsConfig();
-            List<RestEntry> entries = context.getElasticRestCalls();
+            List<RestEntry> entries = new ArrayList();
```

Comment:
```suggestion
            List<RestEntry> entries = new ArrayList<>();
```

Reply:
Weird, not getting a warning on this. Fixed though.